packages/kbn-optimizer/src/optimizer/optimizer_cache_key.test.ts

```diff
@@ -93,6 +93,8 @@ describe('getOptimizerCacheKey()', () => {
           "themeTags": Array [
             "v8light",
             "v8dark",
+            "borealislight",
```

Comment:
nit: why not `v9light`...?

Reply:
@JasonStoltz We didn't use `Amsterdam` for v8, perhaps we should use `v9` here...?

Reply:
In the past, themes were pretty much directly related to Kibana major versions. Moving forward, we want themes to be more dynamic than that, meaning a theme may change over the lifespan of a single major, and there might be more than one theme available.